testing/benchmarking/automated-benchmark/README.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
<p>Side remark - as we know now this are also required for GKE cluster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>